### ISSUE TITLE
loss is increasing very quickly and stops!

### ISSUE THREAD

**Issue by gkszg** [keras-team/keras#12366]
Hello everyone,
I have a problem when I try to train my network using triplet data (positive, negative and anchor). Also, I am using a standard triplet loss. I used to shuffle data or I changed my batch size. Unfortunately, my result is like that: 
I have no idea why loss stops changing. 
![loss](https://user-images.githubusercontent.com/11995514/53569713-e161b480-3b75-11e9-851c-e00239c04d45.png)

Thanks for in advance.
